src/ontodia/diagram/model.ts

```diff
@@ -47,14 +47,14 @@ export class DiagramModel {
     private linkFetching: DataFetchingThread;
     private propertyLabelFetching: DataFetchingThread;
 
-    constructor(isViewOnly = false) {
-        this._isViewOnly = isViewOnly;
+    constructor(hideHalo = false) {
+        this._hideHalo = hideHalo;
```

Comment:
We already have `disableDefaultHalo` option in `DiagramView`. Should we use it instead and just remove `isViewOnly`?